fl4health/utils/warmed_up_module.py

```diff
@@ -0,0 +1,90 @@
+import json
+import os
+from logging import ERROR, INFO
+from typing import Optional
+
+import torch
+from flwr.common.logger import log
+
+
+class WarmedUpModule:
+    """This class is used to load a pretrained model into the current model."""
+
+    def __init__(self, pretrained_model_dir: Optional[str], weights_mapping_dir: Optional[str] = None) -> None:
```

Comment:
Add `= None` to `pretrained_model_dir` for consistency.

Reply:
Again, initially I am thinking to not make it optional at all, but this is just go around for smoke test. Once I change it to support getting address for pretrained model in its args, I will fix this.

Reply:
Yeah, it actually makes more sense for it to not be optional.